gems/workflow/app/models/workflow/instance/process.rb

```diff
@@ -23,9 +27,18 @@ def effort
       super || self.definition.effort
     end
 
-    # TODO
     def status
```

Comment:
@keithtom this function will check all the steps are complete, and return the `DONE` status.